e2e/scripts/prepare-server.sh

```diff
@@ -75,7 +75,6 @@ docker run --rm --quiet --name "${COMPOSE_PROJECT_NAME}_curl_callsoffloader" --n
 echo "MM_LICENSE=${MM_PLUGIN_CALLS_TEST_LICENSE}" >>${WORKSPACE}/dotenv/app.private.env
 echo "MM_FEATUREFLAGS_BoardsProduct=true" >>${WORKSPACE}/dotenv/app.private.env
 echo "MM_SERVICEENVIRONMENT=test" >>${WORKSPACE}/dotenv/app.private.env
-echo "MM_CALLS_JOB_SERVICE_URL=http://calls-offloader:4545" >>${WORKSPACE}/dotenv/app.private.env
```

Comment:
Removing this override so it doesn't mess with our expected snapshots of admin console settings.